Database/Corrections/sodQuestFixes.lua

```diff
@@ -1129,6 +1173,250 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410099,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90061] = {
+            [questKeys.name] = "Chaos Bolt",
+            [questKeys.startedBy] = {{202060}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 9,
+            [questKeys.requiredRaces] = raceIDs.ALL_ALLIANCE,
+            [questKeys.requiredClasses] = classIDs.WARLOCK,
+            [questKeys.objectivesText] = {"Gather some Mage and Warlock friends and attack the Frozen Murloc with fire spells."},
+            [questKeys.requiredSpell] = -403925,
+            [questKeys.zoneOrSort] = sortKeys.WARLOCK,
+        },
+        [90062] = {
+            [questKeys.name] = "Chaos Bolt",
+            [questKeys.startedBy] = {{208275}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 9,
+            [questKeys.requiredRaces] = raceIDs.ALL_HORDE,
+            [questKeys.requiredClasses] = classIDs.WARLOCK,
+            [questKeys.objectivesText] = {"Gather some Mage and Warlock friends and attack the Frozen Makrura with fire spells."},
+            [questKeys.requiredSpell] = -403925,
+            [questKeys.zoneOrSort] = sortKeys.WARLOCK,
+        },
+        [90063] = {
+            [questKeys.name] = "Chaos Bolt",
+            [questKeys.startedBy] = {{202060}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 9,
+            [questKeys.requiredRaces] = raceIDs.ALL_HORDE,
+            [questKeys.requiredClasses] = classIDs.WARLOCK,
+            [questKeys.objectivesText] = {"Gather some Mage and Warlock friends and attack the Frozen Murloc with fire spells."},
+            [questKeys.requiredSpell] = -403925,
+            [questKeys.zoneOrSort] = sortKeys.WARLOCK,
+        },
```

Comment:
These can be combined into a single one and simply make the `startedBy` contain both IDs. Like so:

```lua
            [questKeys.startedBy] = {{208275,202060}},
```

Reply:
Even thought the creature is different?
Would you then prefer the objective text to state "Frozen Makrura/Murloc"?

Reply:
Ah. I missed that one. Then this is totally fine, thank you :D